Source/src/core/preferences/src/EditorPreferences.h

```diff
@@ -103,15 +103,15 @@ namespace Hachiko
 
     private:
         bool display_debug_draw = false;
-        float3 scene_background_color = float3(0.9f);
 
         //TODO remove this property in favor of the camera component
         bool display_camera_settings = false;
 
         float max_fps = 250.0f;
         bool fullscreen = false;
         bool scene_autosave = false;
-        int vsync = 0;
+        bool vsync = true;
```

Comment:
vsync could be -1, 0, 1, by setting it as bool we remove the posibility to have adaptive vsync
https://wiki.libsdl.org/SDL_GL_SetSwapInterval

Reply:
Yes, I know,  but here is handled as a bool in ModuleWindow / WindowConfiguration, so I change it for consistency.